domino/routes.py

```diff
@@ -61,11 +59,11 @@ def runs_list(self):
     def runs_start(self):
         return self._build_project_url() + "/runs"
 
-    def runs_status(self, runId):
-        return self._build_project_url() + "/runs/" + runId
+    def runs_status(self, run_id):
+        return self._build_project_url() + "/runs/" + run_id
```

Comment:
Please don't change this as it could be called by clients as a keyword argument, meaning this is a breaking change
`dom.runs_status(runId="foo")`

Reply:
Or accept both and prefer `run_id`.